scripts/chart-test.sh

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env bash
 
+# Enforce version bump on exporters
+
+# TODO DEBUG
+git rev-parse --abbrev-ref HEAD
+git --no-pager diff master... --name-status exporters/
+# TODO DEBUG
+
+if git status exporters | grep exporters &> /dev/null || git --no-pager diff master... --name-status exporters/ | grep exporters &> /dev/null; then
+  EXPORT_VERSION_IN_MASTER="$(curl https://raw.githubusercontent.com/dora-metrics/pelorus/master/exporters/setup.py &> /dev/null)"
+  CURRENT_EXPORT_VERSION="$(grep '    version="' exporters/setup.py  | cut -c 14- | rev | cut -c 3- | rev)"
+  if $EXPORT_VERSION_IN_MASTER | grep "$CURRENT_EXPORT_VERSION" &> /dev/null; then
+    # TODO breaks when updating from 2.0.10-rc.1 to 2.0.10, for example
```

Comment:
I think it can be so much easier.

We do have a script:
```
$ make rc-release (runs $./scripts/create_release_pr.sh -n)
```

Which is doing all the version checking and updating logic, so why duplicate it here. It also checks if the PR is rebased on master, this way we don't need to do the curl against master/exporters/setup.py just check if one of the chart file versions is updated, when the exporters/* files were modified.

```
charts/pelorus/subcharts/exporters/Chart.yaml
```

The rest will happen as there is part which checks if the versions are the same across other Chart.yaml files, operator files etc...

Reply:
how about this?

```sh
if git status exporters | grep exporters &> /dev/null || git --no-pager diff $REMOTE/master --name-status exporters/ | grep exporters &> /dev/null; then
  CURRENT_CHART_VERSION="$(grep '^version: ' charts/pelorus/Chart.yaml  | cut -c 10-)"
  CHART_FILE_IN_MASTER="$(curl https://raw.githubusercontent.com/dora-metrics/pelorus/master/charts/pelorus/Chart.yaml 2> /dev/null)"
  CHART_VERSION_IN_MASTER=$(echo "$CHART_FILE_IN_MASTER" | grep '^version: ' | cut -c 10-)
  if [ "$CHART_VERSION_IN_MASTER" == "$CURRENT_CHART_VERSION" ]; then
    echo "ERROR: Exporters were modified, Charts need version bumping!"
    exit 1
  fi
fi
```

with this we do not need to add version in exporters